lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -143,7 +147,9 @@ import Web.HttpApiData
 
 import qualified Data.Aeson as Aeson
 import qualified Data.Aeson.Types as Aeson
+import qualified Data.ByteString.Base64 as B64
```

Comment:
For consistency, and for one less direct dependency, you can use `Data.ByteArray.Encoding.convertFromBase Base64`. The only problem with this function is that it's polymorphic on both ends - very slippery.

Reply:
I promise to look at this in the second PR. now will leave it as it is as it seems to work properly